common/lib/xmodule/xmodule/tests/test_library_content.py

```diff
@@ -74,18 +74,14 @@ class TestLibraryContentExportImport(LibraryContentTest):
     """
     Export and import tests for LibraryContentBlock
     """
+    def setUp(self):
+        super().setUp()
 
-    maxDiff = None
-
-    def test_xml_export_import_cycle(self):
-        """
-        Test the export-import cycle.
-        """
         # Children will only set after calling this.
         self.lc_block.refresh_children()
-        lc_block = self.store.get_item(self.lc_block.location)
+        self.lc_block = self.store.get_item(self.lc_block.location)
 
-        expected_olx = (
+        self.expected_olx = (
```

Comment:
Why are these two variables changed to attributes?  They don't look like they are needed beyond the method.